custom_model_runner/datarobot_drum/resource/components/Python/prediction_server/prediction_server.py

```diff
@@ -209,3 +212,24 @@ def terminate(self):
         terminate_op = getattr(self._predictor, "terminate", None)
         if callable(terminate_op):
             terminate_op()
+
+    def load_flask_extensions(self, app):
+        custom_file_paths = list(Path(self._code_dir).rglob("{}.py".format(FLASK_EXT_FILE_NAME)))
+        assert len(custom_file_paths) <= 1
```

Comment:
It would be nice to have a useful user facing message for this.

Reply:
Yeah, that makes sense. I copied the logic from the `custom.py` hook code and didn't think much about it but I also don't like using `assert` to catch bad user input. I'll go update that code as well: https://github.com/datarobot/datarobot-user-models/blob/master/custom_model_runner/datarobot_drum/drum/model_adapter.py#L139-L141

Reply:
Thanks